src/Umbraco.Web.UI.Client/src/packages/documents/documents/collection/action/create-document-collection-action.element.ts

```diff
@@ -56,20 +56,20 @@ export class UmbCreateDocumentCollectionActionElement extends UmbLitElement {
 
 	override async firstUpdated() {
```

Comment:
Ideally this is not handled on firstUpdated, instead I would suggest observing both `workspaceContext.unique` and  `workspaceContext.contentTypeUnique` in one Observe..

this style: `this.observe(observeMultiple([workspaceContext.unique, workspaceContext.contentTypeUnique]), ...`

And then once both have a value call the `this.#retrieveAllowedDocumentTypesOf`

Reply:
These points make sense @nielslyngsoe - but you are commenting on lines and patterns that weren't part of this PR.  I'm probably not the person best placed to refactor so if it's OK with you, can I leave to you to amend if you see fit to do so please?